src/python/pants/util/docutil.py

```diff
@@ -21,3 +23,11 @@ def doc_url(slug: str) -> str:
 def git_url(fp: str) -> str:
     """Link to code in pantsbuild/pants."""
     return f"https://github.com/pantsbuild/pants/blob/release_{PANTS_SEMVER}/{fp}"
+
+
+def bin_name() -> str:
+    """Return the Pants binary name.
+
+    (E.g. "./pants")
+    """  # noqa: PANTSBIN
+    return os.environ.get("PANTS_BIN_NAME", "./pants")  # noqa: PANTSBIN
```

Comment:
This should probably raise `KeyError` if not set, right? We want to be made aware right away if `local_pants_runner.py` ever breaks in not setting the value.

Reply:
Ok made this work with a little import shuffling